### ISSUE TITLE
test_tarfile: test_extractall_symlinks() fails on Windows with: [WinError 206] The filename or extension is too long

### ISSUE THREAD

**Issue by vstinner** [python/cpython#81709]
BPO | [37528](https://bugs.python.org/issue37528)
--- | :---
Nosy | @pfmoore, @vstinner, @tjguk, @zware, @eryksun, @zooba, @pablogsal

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = None
created_at = <Date 2019-07-09.11:34:17.760>
labels = ['tests', '3.9', 'OS-windows']
title = 'test_tarfile: test_extractall_symlinks() fails on Windows with: [WinError 206] The filename or extension is too long'
updated_at = <Date 2019-08-22.09:39:08.645>
user = 'https://github.com/vstinner'
```

bugs.python.org fields:
```python
activity = <Date 2019-08-22.09:39:08.645>
actor = 'vstinner'
assignee = 'none'
closed = False
closed_date = None
closer = None
components = ['Tests', 'Windows']
creation = <Date 2019-07-09.11:34:17.760>
creator = 'vstinner'
dependencies = []
files = []
hgrepos = []
issue_num = 37528
keywords = []
message_count = 5.0
messages = ['347549', '347550', '347597', '350143', '350177']
nosy_count = 7.0
nosy_names = ['paul.moore', 'vstinner', 'tim.golden', 'zach.ware', 'eryksun', 'steve.dower', 'pablogsal']
pr_nums = []
priority = 'normal'
resolution = None
stage = None
status = 'open'
superseder = None
type = None
url = 'https://bugs.python.org/issue37528'
versions = ['Python 3.9']
```

</p></details>

**Comment by vstinner** [comment]
While it would be nice to fix this real bug, I don't have the bandwidth to work on this issue, so I just close it.

**Comment by pfmoore** [comment]
If it’s a genuine bug why not leave it open for someone else to pick up?

**Comment by vstinner** [comment]
The issue is about a buildbot failure. But Windows buildbots pass for months: the test_tarfile doesn't occur anymore. Maybe buildbots were upgraded to Windows 10 or Windows 11. Maybe paths used on Windows got shorter. This is not activity for 3 years, so I understand that nobody cares, or the bug was just fixed in the meanwhile.

More generally, I have limited bandwidth to fix issues specific to Windows, and yesterday I closed most old issues that I opened to help me tracking my "TODO lists": https://github.com/python/cpython/issues/vstinner. I had 3 pages, now it's only 1 page. It's easier to manage :-)

I don't know how to "reassign" an issue to someone else, so it's no longer in my https://github.com/python/cpython/issues/vstinner list.

Note: recently I fixed another bugs about long temporary paths on Unix, I found a simple fix, but the scope of the issue was more narrow: https://github.com/python/cpython/issues/93852
